docs/azure/TOC.yml

```diff
@@ -3,77 +3,79 @@
 - name: Get started on Azure with .NET
   expanded: true
   items:
-    - name: Introduction to Azure and .NET
-      href: intro.md
-    - name: Key Azure services for .NET developers
-      href: key-azure-services.md
-    - name: Configure your development environment
-      items: 
-      - name: Create an Azure account
-        href: create-azure-account.md
-      - name: Configure Visual Studio for Azure development (Windows)
-        href: configure-visual-studio.md
-      - name: Configure VS Code for Azure development (cross-platform)
-        href: configure-vs-code.md
-      - name: Install the Azure CLI
-        href: install-azure-cli.md
-      - name: Install additional tools
-        href: azure-tools.md
-      - name: Development setup checklist
-        href: dotnet-dev-env-checklist.md                
+      - name: Introduction to Azure and .NET
```

Comment:
Do we want to reformat all these items?

Reply:
I think VSCode reformatted all the indents when I hit ctrl-s.